board/SConscript

```diff
@@ -127,6 +127,17 @@ else:
   cert_fn = File("../certs/debug").srcnode().abspath
   flags += ["-DALLOW_DEBUG"]
 
+if os.getenv("DEBUG"):
```

Comment:
for loop and define anything prefixed with `DEBUG`?

Reply:
hmm, I feel like it is kind of nice I can look here and discover the list of flags that can be set

Reply:
that’s true. maybe keep them in a list and use a for loop?

Reply:
some of them don't work if you set them globally like this (they cause compile errors) so I removed them from this PR